src/server/cmd/pachd/main.go

```diff
@@ -1,8 +1,10 @@
 package main
 
 import (
+	"flag"
```

Comment:
Could you use pflag instead? It's what we use in other CLIs because it's bundled with cobra. Also it's more posix compliant I think... which seems like a good thing. Just import `github.com/spf13/pflag`, it's already vendored.